src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:38237/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:38237/project/issues?id=sample&pullRequest=6470&issues=AWkpVuu8nFo4g8VoyGkd&open=AWkpVuu8nFo4g8VoyGkd)

[ISSUE_KEY]: AWkpVuu8nFo4g8VoyGkd